ui/userunlock.js

```diff
@@ -109,7 +109,7 @@ function prompt_sign_in() {
             },
             newAccount: {
                 text: 'Create New Account',
-                btnClass: 'btn-blue',
+                btnClass: 'btn-blue create-new-account',
```

Comment:
Why add `create-new-account` in the class here? What does it achieve?

Reply:
The `create-new-account` and `sign-in` classes are used to identify the buttons, so they can be referenced and clicked during tests. Do you think an `id` attribute would be more appropriate?

Reply:
yeah I guess id would be better, since the `btnClass` is used by the pop up module I am using just to determine how the button looks.

Reply:
@LefterisJP what property here would translate to an `id` attribute?